librz/cons/input.c

```diff
@@ -109,7 +109,13 @@ RZ_API int rz_cons_arrow_to_hjkl(int ch) {
 			ch = rz_cons_readchar();
 		}
 #else
-		ch = 0xf1 + (ch & 0xf);
+		switch (ch) { // Arrow keys
+		case 'A': ch = 'k'; break;
+		case 'B': ch = 'j'; break;
+		case 'C': ch = 'l'; break;
+		case 'D': ch = 'h'; break;
```

Comment:
is this CTRL-D? Because afaik CTRL-D has a special meaning in rizin/shell.

Reply:
No, the full sequence is actually `\x27 \x4f D` (or `^[OD`).

Reply:
@ret2libc I have opened the same PR in the radare2 project (22123) and it turns out the key codes sent are caused by my terminal being stuck in _application mode_, which seems to be a configuration issue (especially if no one but me has seen this problem).

The ^L bug, I think, is still real.  Please let me know if you can reproduce and if you want me to open a separate PR.

Here are the steps to reproduce the Ctrl-L issue:
- launch `rizin`
- type a string, e.g. "foobar"
- move the cursor to the left, e.g. ^A to go to the first column
- hit ^L one or several times

Reply:
> Here are the steps to reproduce the Ctrl-L issue:
> 
> * launch `rizin`
> * type a string, e.g. "foobar"
> * move the cursor to the left, e.g. ^A to go to the first column
> * hit ^L one or several times

I can reproduce this 👍  and it is fixed in your PR